src/components/user-defined-sets/NamedSetManager.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable jsx-a11y/accessible-emoji */
+import React, { useState } from 'react';
+import * as Sets from './sets';
+
+export default function NamedSetManager(props) {
+  const {
+    sets,
+    name,
+    onUpdateSets = (msg) => {
+      console.warn(`onUpdateSets from NamedSetManager ${msg}`);
+    },
+  } = props;
+  const set = sets.namedSets.get(name);
+  if (!set) {
+    return null;
```

Comment:
Would there be any valid reason to reach this point? Maybe a console.warn?

Reply:
No, I will replace with a console.warn